src/main/java/frc/robot/vision/VisionConstants.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.vision;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class VisionConstants {
+
+	public static final String FILTERED_ESTIMATION_LOGPATH_ADDITION = "FilteredEstimation/";
+
+	public static final String NON_FILTERED_ESTIMATION_LOGPATH_ADDITION = "NonFilteredEstimation/";
+
+	public static final String FILTERED_OUT_RAW_DATA_LOGPATH_ADDITION = "FilteredOutRawData/";
+
+	public static final String SOURCE_LOGPATH_ADDITION = "VisionSource/";
+
+
+	public static final int LIMELIGHT_ENTRY_ARRAY_LENGTH = 7;
```

Comment:
not true for `aprilTagPoseArray` but just for `robotPoseArray`. I think we should cut both of them down to 6 since to fix the inconsistency in the Limelight's interface (anyway we can just add cl+tl). 

[Context](https://docs.limelightvision.io/docs/docs-limelight/apis/complete-networktables-api)